src/schema/schema.graphql

```diff
@@ -2799,6 +2799,19 @@ type GetTrustedDevicesResponse {
 	thisDeviceTrusted: Boolean
 }
 
+type Device {
+	id: ID
+	name: String
+	firstReqReceived: Int
+	lastPolicyUpdate: Int
+	lastUsedDate: String
+	protocol: Float
+	provisionable: Boolean
+	status: Int
```

Comment:
we should have enum for this

Reply:
I had tried but failed to do so as we are getting Byte/numbers from server, keeping this as it is.